src/software/pipeline/main_LdrToHdrCalibration.cpp

```diff
@@ -382,5 +444,24 @@ int aliceVision_main(int argc, char** argv)
     response.write(outputResponsePath);
     response.writeHtml(htmlOutput, "response");
 
+    double meanIndex = 0.0;
+    for (int i = 0; i < refIndexes.size(); i++)
+    {
+        meanIndex += static_cast<double>(refIndexes[i]);
+    }
+    const int refIndex = static_cast<int>(std::round(meanIndex /= refIndexes.size()));
+
+    const std::string expRefIndexFilename = (fs::path(outputResponsePath).parent_path() / (std::string("exposureRefIndex") + std::string(".txt"))).string();
+    std::ofstream file(expRefIndexFilename);
+    if (!file)
+    {
+        ALICEVISION_LOG_WARNING("Unable to create file " << expRefIndexFilename << " for storing the exposure reference Index");
```

Comment:
Warning is sufficient because in the node that reads the file (merging node) the file is not mandatory, the user can set a manual parameter. In case the file is required due to an out of range setting of the manual parameter, an error is thrown if the file cannot be read.

Reply:
A file system error should generate an explicit error.
So we should throw an error here, as we don't want an automatic fallback (== a change in the behavior of the software) due to a file system error.

Reply:
It is not an automatic fallback. This file system error generates an explicit error but this explicit error is returned by the merging node, only if the file is needed. If the user wants to use the manual setting available in the merging node by setting it to the correct range of values, he will not be able to do so because the previous node will have thrown an error.

Reply:
If there is a permission error here for instance, it should stop the process in all case, isn't it?